packages/swing-store/src/swingStore.js

```diff
@@ -82,6 +87,154 @@ export function makeSnapStoreIO() {
  * }} SwingStore
  */
 
+/**
+ * @typedef {[
+ *   key: string,
+ *   value: string|null,
```

Comment:
It came up that golang is a stupid language, and using `undefined` for deletion would make my life much easier
```suggestion
 *   value?: string | undefined,
```

Reply:
> It came up that golang is a stupid language

I'm shocked.  Shocked, I say.

Since our choice of `null` here was pretty arbitrary anyway, switching to `undefined` is fine with me.